db/models.py

```diff
@@ -17,7 +18,7 @@ def __str__(self) -> str:
 
 
 class Movie(models.Model):
-    title = models.CharField(max_length=255)
+    title = models.CharField(max_length=255, db_index=True)
```

Comment:
I think its bad practice when you use db_index=True. Because when the model expands it will be difficult to maintain

Reply:
also related_name